xml/System.Formats.Cbor/CborConformanceMode.xml

```diff
@@ -34,7 +34,15 @@
       </ReturnValue>
       <MemberValue>2</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Ensures that the CBOR data is canonical, as specified in RFC7049 section 3.9.
+              Extends strict conformance with the following requirements:
+              <list type="bullet">
+                <item>Integers must be encoded as small as possible.</item>
+                <item>Maps (major type 5) must contain keys sorted by encoding.</item>
+                <item>Indefinite-length items must be made into definite-length items.</item>
+              </list>
+            </summary>
```

Comment:
nit: indentation is off by one tab here and in the other summaries.

Reply:
```suggestion
        <summary>
          <para>Ensures that the CBOR data is canonical, as specified in RFC7049 section 3.9.</para>
          <para>Extends strict conformance with the following requirements:</para>
            <list type="bullet">
              <item><description>Integers must be encoded as small as possible.</description></item>
              <item><description>Maps (major type 5) must contain keys sorted by encoding.</description></item>
              <item><description>Indefinite-length items must be made into definite-length items.</description></item>
          </list>
        </summary>
```